### PR TITLE
Fixes know postgres issues.

### PR DESCRIPTION
**Pull request by demianfe**

### CODE REVIEW

File: ormin/ormin_postgre.nim [hunk 1]
```diff
@@ -11,6 +11,7 @@ type
   varchar* = string
   integer* = int
   timestamp* = string
+  uuid* = string
```
Comment (Araq) [inline ormin/ormin_postgre.nim]: Unused.

File: ormin/ormin_postgre.nim [hunk 2]
```diff
@@ -25,15 +26,17 @@ proc c_strtod(buf: cstring, endptr: ptr cstring = nil): float64 {.
...
+var sid = 0
+
+template genId*: untyped =
+  inc(sid)
+  sid
 
 proc prepareStmt*(db: DbConn; q: string): PStmt =
-  static:
-    inc sid
-    const name = "ormin" & $sid
```
Comment (Araq) [inline ormin/ormin_postgre.nim]: I think the ``const`` idea can be kept if you make it a ``.compileTime`` var and avoid the ``static`` block.

File: ormin/ormin_postgre.nim [hunk 3]
```diff
@@ -25,15 +26,17 @@ proc c_strtod(buf: cstring, endptr: ptr cstring = nil): float64 {.
...
-  result = cstring(name)
+  var name = "ormin" & $genId
+  result = name
```
Comment (Araq) [inline ormin/ormin_postgre.nim]: Why not ``result = "ormin" & $genId`` ?

File: ormin/ormin_postgre.nim (hunk 1 above)
Comment (demianfe) [reply ormin/ormin_postgre.nim]: Very much used: [Postgres UUID Type](https://www.postgresql.org/docs/11/datatype-uuid.html)
Without it the following error message appears: 
`/ormin/ormin/queries.nim(141, 17) Error: undeclared identifier: 'uuid'`

File: ormin/ormin_postgre.nim (hunk 2 above)
Comment (demianfe) [reply ormin/ormin_postgre.nim]: If the const is kept the error is the following: 
`/ormin/ormin/ormin_postgre.nim(38, 15) Error: cannot 'importc' variable at compile time; pqprepare`

If let, or var is used instead the error is:
`/ormin/queries.nim(809, 43) Error: request to generate code for .compileTime proc: prepareStmt`

File: ormin/ormin_postgre.nim (hunk 1 above)
Comment (Araq) [reply ormin/ormin_postgre.nim]: ok.

**Pull request closed without merging**